cli/src/trades.rs

```diff
@@ -67,3 +70,67 @@ pub fn all(client: &Client, matches: &ArgMatches) -> Result<()> {
     });
     res
 }
+
+pub fn aggregations(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+
+    let endpoint = {
+        let base_asset_type = matches
```

Comment:
Are there other apis that require extracting asset identifiers?

if so, i'm wondering if we could refactor this into a utils function or a macro where I provide a 3 string tuple, one for each field (`type`, `code`, `issuer`). It would also take a ArgMatches and it would return the result from asset identifier.

brainstorming...

```rust
pub struct Fields<'a>(pub &'a str, pub &'a str, pub &'a str);

pub fn get_identifier(matches: &ArgMatches, Fields(type, code, issuer): Fields) -> Result<AssetIdentifier>;
```

An usage might look like:
```
use asset_identifier::{Fields, get_identifier};

let base = get_identifier(
    &matches,
    Fields("base_asset_type", "base_asset_code", "base_asset_issuer")
)?;
```